src/backend/gl/src/command.rs

```diff
@@ -326,44 +327,50 @@ impl RawCommandBuffer {
 
     fn update_blend_targets(&mut self, blend_targets: &Vec<pso::ColorBlendDesc>) {
```

Comment:
should be `&[pso::ColorBlendDesc]`

Reply:
yeah, definitely, I can change that with this PR